### PR TITLE
MLUI-139: new-entity.component.ts

### PR DESCRIPTION
**Pull request by brucean52**

### CODE REVIEW

File: quick-start/src/main/ui/app/shared/stories/components/new-entity/new-entity.component.stories.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  @Output() createClicked = new EventEmitter();
+  constructor(
+    private dialogService: MdlDialogService
+  ) { }
+  openModal() {
+    this.dialogService.showCustomDialog({
+      component: NewEntityComponent,
+      providers: [
+        { provide: 'actions', useValue: {save: () => {
+          this.create();
```
Comment (alexsmr) [inline quick-start/src/main/ui/app/shared/stories/components/new-entity/new-entity.component.stories.ts]: No need to create the function here, just execute:
```suggestion
           this.createClicked.emit()
```

File: quick-start/src/main/ui/app/shared/stories/components/new-entity/new-entity.component.stories.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        }}}
+      ],
+      isModal: true
+    });
+  }
+  create() {
+    this.createClicked.emit();
+  }
```
Comment (alexsmr) [inline quick-start/src/main/ui/app/shared/stories/components/new-entity/new-entity.component.stories.ts]: No need above function

**Comment by alexsmr** [comment]
There is a build failure:
```
Cannot determine the module for class DialogButtonComponent in <https://jenkins.marklogic.com/job/dhf-QS-develop-rh7-Commit/ws/quick-start/src/main/ui/app/shared/stories/components/new-entity/new-entity.component.stories.ts!> Add DialogButtonComponent to the NgModule to fix it.
```

**Comment by brucean52** [comment]
> There is a build failure:
> 
> ```
> Cannot determine the module for class DialogButtonComponent in <https://jenkins.marklogic.com/job/dhf-QS-develop-rh7-Commit/ws/quick-start/src/main/ui/app/shared/stories/components/new-entity/new-entity.component.stories.ts!> Add DialogButtonComponent to the NgModule to fix it.
> ```



> There is a build failure:
> 
> ```
> Cannot determine the module for class DialogButtonComponent in <https://jenkins.marklogic.com/job/dhf-QS-develop-rh7-Commit/ws/quick-start/src/main/ui/app/shared/stories/components/new-entity/new-entity.component.stories.ts!> Add DialogButtonComponent to the NgModule to fix it.
> ```

How would you recommend I resolve this? I could include add it as a declaration in the utils.module.ts NgModule.

**Comment by alexsmr** [comment]
Just create a new module in the story.

**Pull request merged**